test/test_enclave_wrangler/test_objects_api.py

```diff
@@ -280,6 +276,93 @@ def test_get_csets_over_threshold(self):
         result = get_csets_over_threshold(csets, 30, 'cset_ids')
         self.assertEqual(result, {1, 2})
 
+    @classmethod
+    def _fetch_csets_from_db(
+        cls, limit=2, cset_ids: List[int] = None, convert_to_enclave_object=True
+    ) -> Dict[int, Dict]:
+        """Fetch some sample code sets from the database.
+
+        :param: limit: If not cset_ids, will fetch n 'limit' csets; just want to make sure multiple sets work"""
+        cset_id_str: str = ', '.join([str(x) for x in cset_ids]) if cset_ids else ''
+        # TODO: test_n3c shouldn't be necessary, but GH action is failing.
+        query: str = f"""SELECT * FROM code_sets WHERE codeset_id IN ({cset_id_str});""" if cset_ids \
+            else f"""SELECT * FROM code_sets LIMIT {limit};"""
+        csets: Dict[int, Dict] = {x['codeset_id']: dict(x) for x in sql_query(cls.con, query)}
+        if convert_to_enclave_object:
+            csets = {k: convert_row('code_sets', 'OMOPConceptSet', v) for k, v in csets.items()}
+        return csets
+
+    def test_update_cset_metadata_from_objs(self):
```

Comment:
## `test_update_cset_metadata_from_objs()`
1. Does arbitrary mutations on all fields, and updates row in DB.
2. Fetches from enclave and updates again.